src/main/java/com/openlist/app/Utils/AppleMusicTokenGenerator.java

```diff
@@ -0,0 +1,55 @@
+package com.openlist.app.util; 
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.algorithms.Algorithm;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.security.GeneralSecurityException;
+import java.security.KeyFactory;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.PKCS8EncodedKeySpec;
+import java.time.Instant;
+import java.util.Date;
+import java.security.NoSuchAlgorithmException;
+import java.security.interfaces.ECPrivateKey;
+import java.util.Base64;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import com.openlist.app.config.AppleMusicConfig;
+
+@Component
+public class AppleMusicTokenGenerator {
```

Comment:
Don't worry toooo much about this file. It's basically doing some complex Apple required stuff where they want us to take our apple IDs and tokens and running them through complex encryption functions to let us 'identify' ourselves when hitting the API. We won't really need to change this and it's just using this function when building the headers for the Apple Music API.